### PR TITLE
lsp-plugins: split subpkgs

### PR DESCRIPTION
**Pull request by dkwo**
Tested by me and @slimjimsoftware 

This helps unclutter things. @leahneukirchen can you take a look?

### CODE REVIEW

**Comment by classabbyamp** [comment]
why is this needed? (how is it currently "cluttered"?)

if you're going to do this, you need to make the main package depend on everything you split out to not break other people's setups

**Comment by slimjimsoftware** [comment]
@classabbyamp desktop menus (for some desktops) get cluttered with LSP icons, there are numerous reports of this and it appears to be something to do with GNOME not following XDG standards? Anyway a quick google shows up this:

https://www.linuxquestions.org/questions/linux-general-1/my-multimedia-menu-is-full-of-some-audio-related-stuff-any-idea-where-these-came-from-4175721036/

https://www.reddit.com/r/linux4noobs/comments/g40e3a/unwanted_lsp_plugins_showing_up/

I can confirm this happens on plasma 6 too. 

Apple Silicon mac support requires just the lv2 set of lsp-plugins to enable audio so rather than install the entire set, we can split into subpackages and also avoid the icon clutter

File: srcpkgs/lsp-plugins/template [hunk 1]
```diff
@@ -21,6 +21,55 @@ do_configure() {
 	make LD="$CXX" PREFIX=/usr config
 }
 
+lsp-plugins-lib_package() {
```
Comment (classabbyamp) [inline srcpkgs/lsp-plugins/template:24]: standard package naming would be `-libs`

**Review by classabbyamp — COMMENTED** [review]
`.a` static libs should be in the devel pkg

File: srcpkgs/lsp-plugins/template (hunk 1 above)
Comment (dkwo) [reply srcpkgs/lsp-plugins/template:24]: Done.

**Comment by dkwo** [comment]
Typically, one only needs one plugin, not necessarily all of them, as well as icons etc. You can compare the clutter via `xls`.

If the main package depends on all the splitted ones, then what's the point? I propose an install message, similar to Alpine's one. 

> The plugins were split up into multiple subpackages [...] Be sure to install ones you were using.

Would this work?

I'm trying to get the devel part correct. Does it look right now?

**Comment by classabbyamp** [comment]
> If the main package depends on all the splitted ones, then what's the point?

to not break people's setups

**Comment by dkwo** [comment]
Let me know if this is now acceptable, then I'll squash my commits.

**Comment by classabbyamp** [comment]
squash it

**Comment by dkwo** [comment]
Done.

**Pull request merged**